src/sql/command.py

```diff
@@ -88,4 +87,22 @@ def result_var(self):
         return self.parsed["result_var"]
 
     def _var_expand(self, sql, user_ns, magic):
-        return Template(sql).render(user_ns)
+        sql = Template(sql).render(user_ns)
+        parsed_sql = magic.shell.var_expand(sql, depth=2)
+
+        has_SQLAlchemy_var_expand = ":" in sql and any(
+            (":" + ns_var_key in sql for ns_var_key in user_ns.keys())
+        )
+        # has_SQLAlchemy_var_expand: detect if using Sqlalchemy fashion - :a
+
+        msg = (
+            "Variable substitution with $var and {var} has been "
+            "deprecated and will be removed in a future version. "
+            "Use {{var}} instead. To remove this, see: "
+            "https://jupysql.ploomber.io/en/latest/howto.html#ignore-deprecation-warnings"  # noqa
+        )
+
+        if parsed_sql != sql or has_SQLAlchemy_var_expand:
+            warnings.warn(msg, FutureWarning)
+
+        return parsed_sql
```

Comment:
this looks unrelated to the PR

Reply:
maybe added by mistake?

Reply:
You right, some merge error
FIx in [a619224](https://github.com/ploomber/jupysql/pull/293/commits/a619224681751aa113b43f073828002f6f2499cd)